auth/index.html

```diff
@@ -1274,6 +1274,12 @@ <h2>Message log</h2>
        */
       var onGetEmbeddedPublicKey = async function (requestId) {
         var embeddedKeyJwk = await TKHQ.getEmbeddedKey();
+
+        // If we don't have an embedded key, return an empty string as the public key
+        if (!embeddedKeyJwk) {
```

Comment:
Need to return early if `embeddedKeyJwk` is null; otherwise, the subsequent lines will error out, and no response will be passed back to the client (aka `iframe-stamper`), leading to a hanging/dangling request